.github/workflows/tests.yml

```diff
@@ -59,3 +59,4 @@ jobs:
         run: |
           ./make.sh
           ruby scripts/update_suite.rb -m
+          gobopt='--disable ana.opt.hashcons' ruby scripts/update_suite.rb -m
```

Comment:
One should probably put these into separate groups with appropriate titles, so one can see fast if it failed or nor